app/uk/gov/hmrc/customs/declaration/connectors/UpscanInitiateConnector.scala

```diff
@@ -35,18 +38,26 @@ class UpscanInitiateConnector @Inject()(http: HttpClient,
 
   private val configKey = "upscan-initiate"
 
-  def send[A](payload: UpscanInitiatePayload, apiVersion: ApiVersion)(implicit vupr: ValidatedUploadPayloadRequest[A]): Future[HttpResponse] = {
+  def send[A](payload: UpscanInitiatePayload, apiVersion: ApiVersion)(implicit vupr: ValidatedUploadPayloadRequest[A]): Future[InitiateUpscanResponsePayload] = {
     val config = Option(serviceConfigProvider.getConfig(s"${apiVersion.configPrefix}$configKey")).getOrElse(throw new IllegalArgumentException("config not found"))
     post(payload, config.url)
   }
 
+
   private def post[A](payload: UpscanInitiatePayload, url: String)(implicit vupr: ValidatedUploadPayloadRequest[A]) = {
 
     implicit val hc: HeaderCarrier = HeaderCarrier()
```

Comment:
our normal pattern is to put Json formats into the companion object of the model class, and put this next to class in the same scala source file. Then we import the object in the client code which brings the implicit into scope eg
```
case class SomeModel(foo: String)
object SomeModel {
  implicit val format = Json.format[SomeModel]
}
```

Reply:
:+1: